app/styles/main.scss

```diff
@@ -154,13 +182,13 @@ $homepageTileHeight : 250px;
   font-size: 1.2em;
 }
 
-.homepageTile:hover {
+.homepageTile:hover, .court-search-bar:hover {
   border-color: $blue;
   cursor: pointer;
 }
 
-.homepageTile.unselected {
-  opacity: 0.5;
+.homepageTile.unselected, .court-search-bar.unselected {
+  opacity: 0.7;
```

Comment:
You just didn't like the opacity so you decided to change it? I'm sure it's not a big difference but the point was to keep eyes focused on the selection and to really make it pop. We can look at this further when we polish everything.